src/Drivers/CMakeLists.txt

```diff
@@ -71,11 +71,6 @@ if(HIOP_USE_MPI AND HIOP_SPARSE)
         nlpPriDec_ex9_sparse_raja.cpp 
         nlpPriDec_ex9_sparse_raja_driver.cpp 
         PROPERTIES LANGUAGE CUDA)
-    elseif(HIOP_USE_HIP)
```

Comment:
Why should we remove this `elseif` section for HIP?

Reply:
Because we already link against HIP device libraries in `cmake/FindHiopHipLibraries.cmake`, which is all that's actually needed for compiling HIP. It's unclear what the CMake best practice will be for this sort of thing, but at least from discussions on Spack and AMD slack channels, it isn't needed.

Reply:
Similarly, can we remove the previous section about `HIOP_USE_CUDA` since we have already link against CUDA via `cmake/FindHiopCUDALibraries.cmake`?

Reply:
No, we probably want to use the CUDA CMake language since there are additional steps with CUDA C++. For example, we want to link with Umpire's device code using the nvidia compiler, and that will only happen if the source language is CUDA.